src/sagemaker/jumpstart/types.py

```diff
@@ -1001,12 +1004,19 @@ def to_json(self) -> Dict[str, Any]:
                             json_obj[att].append(obj.to_json())
                         else:
                             json_obj[att].append(obj)
+                elif isinstance(cur_val, dict):
+                    json_obj[att] = {}
+                    for key, val in cur_val.items():
+                        if issubclass(type(val), JumpStartDataHolderType):
```

Comment:
can we consolidate with this: https://github.com/aws/sagemaker-python-sdk/blob/2f449759b2835eae8552b8d67ef9ff5c55f191e4/src/sagemaker/jumpstart/types.py#L934

Reply:
Sounds good, I'll address in my next PR.